src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+/* reset browser default margins */
+* {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --blue: #00ACDC;
+  --header-heigth: 60px;
+}
+
+header {
```

Comment:
- Don't use tag names for styling (except html and body)
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/dedf016d-4487-4fad-8ab2-01c3b31067fc)